src/escrow/increasing/Lock.sol

```diff
@@ -38,7 +49,7 @@ contract Lock is ILock, ERC721Enumerable, UUPSUpgradeable, DaoAuthorizable, Reen
 
     function supportsInterface(
         bytes4 _interfaceId
-    ) public view override(ERC721Enumerable) returns (bool) {
+    ) public view override(ERC721Enumerable, ERC721URIStorageUpgradeable) returns (bool) {
```

Comment:
Strictly speaking, we should test that the supports interface check is correctly returning as expected for Enumerable vs URI Storage. Of the 2 I think Enumerable is more important to be able to introspect so would want to make sure that is not lost through the override

Reply:
True. I added 2 asserts on `test/escrow/escrow/Lock.t.sol` to verify both interfaces are supported. I also checked and all the parent functions use `|| super.supportsInterface(..)` so they should propagate correctly through the inheritance chain.